### ISSUE TITLE
Add a guide to build an upgradeable ERC20

### ISSUE THREAD

**Issue by facuspagnuolo** [zeppelinos/zos#339]
One of the basic examples we can add to the documentation site is one to show how to build an upgradeable ERC20 token. It may be interesting to add a section of how you can do it using `openzeppelin-eth` to avoid writing it from scratch. 

We can show further examples of minting some tokens on the initialization, upgrading it to anothe version, transferring tokens, transferring control of it, etc

**Comment by ajsantander** [comment]
Isn't this solved already?
See: https://docs.zeppelinos.org/docs/erc20_onboarding.html

**Comment by spalladino** [comment]
@ajsantander no, that guide is on migrating an existing ERC20 token into an upgradeable one. This issue is about creating a new one from scratch, so should be much more simple. Still, we can defer it for later, no need to tackle this issue now.

**Comment by crazyrabbitLTC** [comment]
Is this still something we want to do?

**Comment by spalladino** [comment]
Certainly! Not high prio, but definitely a nice addition to our guides. We could also replace the [linking EVM packages](https://docs.zeppelinos.org/docs/linking.html) guide by this one, to better promote the link between OZ and zOS. Anyway, just a blogpost or forum article is also fine.
